### PR TITLE
Avoid exceptions when the commercial switch is disabled

### PR DESCRIPTION
**Pull request by rich-nguyen**
Make sure the cleaner code doesn't throw an exception in the unlikely event that this switch is off.

## Tested in CODE?
Yes (but missed an error in facia).

### CODE REVIEW

**Comment by rich-nguyen** [comment]
[Error when switching commercial off](https://trello.com/c/hZIxBhjA/122-error-when-switching-commercial-off)

File: common/app/views/support/HtmlCleaner.scala [hunk 1]
```diff
@@ -649,12 +649,14 @@ case class CommercialComponentHigh(isAdvertisementFeature: Boolean, isNetworkFro
...
-      containers.lift(containerIndex).map { case ((container, index)) => {
-          container.after(adSlot)
-          adSlot.wrap(s"""<div class="fc-container fc-container--commercial"></div>""")
-        }
+      for {
+        (container, index) <- containers.lift(containerIndex)
+        slot <- adSlot
+      } {
+          container.after(slot)
+          slot.wrap(s"""<div class="fc-container fc-container--commercial"></div>""")
```
Comment (kelvin-chappell) [inline common/app/views/support/HtmlCleaner.scala]: Don't need the 's'?

File: common/app/views/support/HtmlCleaner.scala [hunk 2]
```diff
@@ -649,12 +649,14 @@ case class CommercialComponentHigh(isAdvertisementFeature: Boolean, isNetworkFro
 
       val adSlotHtml = views.html.fragments.commercial.commercialComponentHigh(isAdvertisementFeature, hasPageSkin)
 
-      val adSlot: Element = Jsoup.parseBodyFragment(adSlotHtml.toString).body().child(0)
+      val adSlot: Option[Element] = Jsoup.parseBodyFragment(adSlotHtml.toString).body().children().toList.headOption
```
Comment (kelvin-chappell) [inline common/app/views/support/HtmlCleaner.scala]: Does jsoup return nulls for empty lists?

**Review by kelvin-chappell — APPROVED** [review]
Just a couple of points.

File: common/app/views/support/HtmlCleaner.scala (hunk 2 above)
Comment (rich-nguyen) [inline common/app/views/support/HtmlCleaner.scala]: no, it throws an exception. :(

**Pull request merged**